pkgs/racket-doc/scribblings/raco/api.scrbl

```diff
@@ -284,7 +284,7 @@ the files that it compiles and produces. The default is @racket[#f].}
 A @racket[#t] value for the parameter causes the compiler to print
 verbose messages about its operations. The default is @racket[#f].}
 
-@defparam[compile-subcollections cols (one-of/c #t #f)]{
```

Comment:
Two things:

1. It would be nice to mention explicitly in the doc below that the argument is expected to be a "true" value or `#f`.
2. I think `(compile-subcollections)` should ideally always return a boolean, similar to parameters like `eval-jit-enabled` (obviously this change is out of scope of your current PR).

Reply:
Actually it seems like *none* of the parameters in `compiler/option` do the Boolean conversion, so the `boolean?` contracts are technically wrong.  I can change them to the correct contracts (and descriptions) *now*, and another PR can address the Boolean conversion part.

Reply:
On a second thought, I can also submit that “another PR” first, so that this can be merged after that :P